### PR TITLE
Cut 0.16.0-rc.0

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,32 +11,53 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
...
+
 ### Fixed
 
 - [#3234](https://github.com/thanos-io/thanos/pull/3234) UI: Fix assets not loading when `--web.prefix-header` is used.
+- [#3184](https://github.com/thanos-io/thanos/pull/3184) Compactor: Fixed support for `web.external-prefix` for Compactor UI.
 
 ### Added
 
-- [#3032](https://github.com/thanos-io/thanos/pull/3032) Query Frontend: Added support for Memacahce cache. Replaced underscores with hyphens in `log_queries_longer_than - > log-queries-longer-than`.
+- [#3032](https://github.com/thanos-io/thanos/pull/3032) Query Frontend: Added support for Memacahce cache.
```
Comment (yeya24) [inline CHANGELOG.md:32]: This should be https://github.com/thanos-io/thanos/pull/3114

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,32 +11,53 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
...
-- [#3154](https://github.com/thanos-io/thanos/pull/3154) Query Frontend: Add metric `thanos_memcached_getmulti_gate_queries_max`.
-- [#3146](https://github.com/thanos-io/thanos/pull/3146) Sidecar: Add `thanos_sidecar_prometheus_store_received_frames` histogram metric.
-- [#3147](https://github.com/thanos-io/thanos/pull/3147) Querier: Add `query.metadata.default-time-range` flag to specify the default metadata time range duration for retrieving labels through Labels and Series API when the range parameters are not specified. The zero value means range covers the time since the beginning.
-- [#3207](https://github.com/thanos-io/thanos/pull/3207) Query Frontend: Add `cache-compression-type` flag to use compression in the query frontend cache.
+- [#3181](https://github.com/thanos-io/thanos/pull/3181) Logging: Added debug level logging for responses between 300-399
+- [#3133](https://github.com/thanos-io/thanos/pull/3133) Query: Allowed passing a `storeMatch[]` to Labels APIs; Time range metadata based store filtering is supported on Labels APIs.
+- [#3154](https://github.com/thanos-io/thanos/pull/3154) Query Frontend: Added metric `thanos_memcached_getmulti_gate_queries_max`.
```
Comment (yeya24) [inline CHANGELOG.md:37]: This is not `Query Frontend`. Should it be `Query`?

Comment (yeya24) [reply CHANGELOG.md:37]: Umm, is it duplicate with https://github.com/thanos-io/thanos/pull/3247/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR49?

**Review by yeya24 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bwplotka) [reply CHANGELOG.md:37]: Is it really duplicate? I think we just have gate for Memcached in query frontend ALSO that's why?

Comment (yeya24) [reply CHANGELOG.md:37]: I don't think so. We are not using the thanos Memcached client in query frontend. We are using the Cortex one.

Comment (bwplotka) [reply CHANGELOG.md:37]: Double checked you are right, not sure how it ended up being here.

**Comment by bwplotka** [comment]
PTAL (:

**Review by yeya24 — APPROVED** [review]
LGTM :rocket:

**Pull request merged**